lib/api.ts

```diff
@@ -180,6 +181,20 @@ export async function getContributorBySlug(file: string, detail = false) {
 
   const calendarData = getCalendarData(weightedActivity.activity);
 
+  const isNewContributor = (() => {
+    const { activity } = activityData;
+
+    const aWeekAgo = sub(new Date(), { days: 7 });
+
+    const firstActivity = activity
+      .filter((act) => act.type === "pr_merged")
+      .sort(
+        (a, b) => new Date(a.time).getTime() - new Date(b.time).getTime(),
+      )[0];
+
+    return isAfter(new Date(firstActivity?.time), aWeekAgo);
```

Comment:
if incase firstActivity is null, it ends up being equivalent to: `new Date()` and that'd always return true when doing `isAfter` right?

```suggestion
    if (!firstActivity) return false;
    return isAfter(firstActivity.time, aWeekAgo);
```

Reply:
not resolved; we don't need the null check in isAfter call right?